src/main/java/pl/xkoem/GameConfiguration.java

```diff
@@ -9,6 +9,14 @@
     private Integer sizeY;
     private String language;
 
+    GameConfiguration() {
+        this(3,3,3);
+    }
+
+    GameConfiguration(Integer sizeX, Integer sizeY, Integer charsToWin) {
+        setBoardSettings(sizeX, sizeY, charsToWin);
+    }
+
     public void setBoardSettings(Integer sizeX, Integer sizeY, Integer charsToWin) throws InvalidParameterException {
         if(charsToWin > sizeX || charsToWin > sizeY) {
             throw new InvalidParameterException("Too big chars to win");
```

Comment:
Tak mi się zdaje, że aktualni indianie to powiedzieliby tak:
"To big ammount of charts to win"
a tak bardziej opowiadająco co to robi, to może:
"Not possible to win on board <X/Y> when ammount of charts to win is specified to <N>"

Reply:
No moze troche lepiej to brzmi